src/PotentialTypes/BasisPotentials/force.jl

```diff
@@ -0,0 +1,11 @@
+function force(A::AbstractSystem, p::BasisPotential)
+    number_of_particles = length(A)
+    d = evaluate_basis_d(A, p.basis_params)
+    f = [zeros(3) for i = 1:number_of_particles]
```

Comment:
for what it's worth, a slightly more Julian way would be `f = fill(zeros(3), length(A))`

Reply:
Changed this to your suggestion. Now that we have a 'Julian' way to do things, we also need a 'Gregorian' way to do things... haha